Add `Beatmap/Pattern/osu!mania` listing and subpages (#8595)
ppy/osu-wiki: wiki/Beatmap/Pattern/Mania/en.md

---
stub: true

> tags:
>   - patterns
>   - osu!mania

Reviewer: ```suggestion
```

---

> # osu!mania Patterns

Reviewer: ```suggestion
# osu!mania patterns
```

> A **pattern** is a small set of [hit objects](/wiki/Hit_object) within a [beatmap](/wiki/Beatmap) arranged in a meaningful way. In osu!mania, this can be done with either **notes**, **long notes**, or a combination of the two.

Reviewer 2: Maybe add the rice/noodles thing here?

Author: I don't really know how to integrate that into this sentence without cluttering it too much, any ideas?

Reviewer 2: Maybe `**notes** (rice)` and similarly to LNs. Not sure if this is needed though.

Author: Think this would just convolute this section, it doesn't seem all that necessary to mention both here when the goal is to keep the explanation simple

> A **pattern** is a small set of [hit objects](/wiki/Hit_object) within a [beatmap](/wiki/Beatmap) arranged in a meaningful way. In osu!mania, this can be done with either **notes**, **long notes**, or a combination of the two.

Reviewer 3: IMO this description needs to include something about commonality, recognisability or the repetitive nature of a pattern - e.g.

A **pattern** is a small set of [hit objects](/wiki/Hit_object) within a [beatmap](/wiki/Beatmap) arranged in a meaningful way. In osu!mania, this can be done with either **notes**, **long notes**, or a combination of the two.
```suggestion
A **pattern** is a common way of arranging a small set of [hit objects](/wiki/Hit_object) within a [beatmap](/wiki/Beatmap). In osu!mania, this can be done with either **notes**, **long notes**, or a combination of the two. In most cases, similar **patterns** are used to represent similar parts of a song.
```

Author: The only issue I have with this is it delves too far into mapping specifics imo, which starts to convolute things. I think a simple explanation for a simple concept would be ideal to avoid any confusion, especially when considering any language barriers. And I think the idea of commonality and recognizability is an inherent trait of a pattern anyway, whether that be in osu! or outside of it.
